app/uid/models.py

```diff
@@ -0,0 +1,87 @@
+from django.db import models
+from neomodel import StructuredNode, StringProperty, DateTimeProperty, BooleanProperty, IntegerProperty, RelationshipTo
+from datetime import datetime
+# from .utils import generate_uid # Import from generate_uid
+# import uuid
+# import hashlib
+
+# Create your models here.
+
+class UIDGenerator:
+    def __init__(self):
+        self.counter = 0
+
+    def generate_uid(self):
+        self.counter += 1
+        return f"UID{self.counter:06d}"  # Zero-padded to 6 digits
```

Comment:
I think the UID's should be formatted like this: 0x00000001. So I think this might be closer to what we want to generate.
```
def generate_uid(self):
    self.counter += 1
    return f"0x{self.counter:08x}